### PR TITLE
modify the document format

### PR DESCRIPTION
**Pull request by jianzhangbjz**
Modify the table format of this doc in order to more readable.

### CODE REVIEW

File: docs/catalog-restrictions.md [hunk 1]
```diff
@@ -62,35 +62,39 @@ subset of properties on service class and service plan resources. The following
 
 `ClusterServiceClass` allowed property names:
 
-| Property Key    | Description    |
-| name |  This key will match the ClusterServiceClass.Name property |
-| spec.externalName | This key will match the ClusterServiceClass.Spec.ExternalName property |
-| spec.externalID | This key will match the ClusterServiceClass.Spec.ExternalID property |
+Property Key    | Description
```
Comment (duglin) [inline docs/catalog-restrictions.md]: For consistency, can you add the `|` around all cells (even the title and `---`) lines?

Comment (jianzhangbjz) [reply docs/catalog-restrictions.md]: Done, thanks!

File: docs/catalog-restrictions.md [hunk 2]
```diff
@@ -62,35 +62,39 @@ subset of properties on service class and service plan resources. The following
 
 `ClusterServiceClass` allowed property names:
 
-| Property Key    | Description    |
-| name |  This key will match the ClusterServiceClass.Name property |
-| spec.externalName | This key will match the ClusterServiceClass.Spec.ExternalName property |
-| spec.externalID | This key will match the ClusterServiceClass.Spec.ExternalID property |
+| Property Key    | Description
```
Comment (duglin) [inline docs/catalog-restrictions.md]: sorry to be picky, but trailing too since all other tables in or docs have those.

Comment (jianzhangbjz) [reply docs/catalog-restrictions.md]: Done.

**Comment by duglin** [comment]
thanks
/lgtm

**Pull request merged**